src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.stories.js

```diff
@@ -0,0 +1,25 @@
+import Carousel from "nuka-carousel";
+import React from "react";
+import BlockQuoteBlock from "./BlockQuoteBlock";
+
+export default {
+  title: "Grid-Aware/BlockQuoteBlock",
+  component: BlockQuoteBlock,
+};
+
+const Template = ({ quote, author }) => (
+  <Carousel renderCenterLeftControls={false} renderCenterRightControls={false}>
+    <BlockQuoteBlock quote={quote} author={author} />
+    <BlockQuoteBlock quote={quote} author={author} />
+    <BlockQuoteBlock quote={quote} author={author} />
+    <BlockQuoteBlock quote={quote} author={author} />
+    <BlockQuoteBlock quote={quote} author={author} />
```

Comment:
Could you make the story display a couple different quotes? Or at least alternate between two quotes so that it feels a bit more realistic?

Reply:
👍 You got it. Pretty sure the code needs some cleaning up though. Please advise.